bin/node/runtime/src/lib.rs

```diff
@@ -663,7 +663,8 @@ impl pallet_election_provider_multi_phase::Config for Runtime {
 	type EstimateCallFee = TransactionPayment;
 	type SignedPhase = SignedPhase;
 	type UnsignedPhase = UnsignedPhase;
-	type SolutionImprovementThreshold = SolutionImprovementThreshold;
+	type SolutionImprovementThresholdUnsigned = SolutionImprovementThreshold;
+	type SolutionImprovementThresholdSigned = GetDefault;
```

Comment:
```suggestion
	type SolutionImprovementThresholdSigned = ();
```

Reply:
Now that we are renaming things, I am a fan of using a shorter name like: `BetterSignedSolution`, `BetterSignedThreshold`, or anything else you can come up with.

Reply:
Also, do you think we can come up with a shorter name? e.g. `BetterSignedThreshold`, `SignedImproveThreshold`